modules/microshift-storage-classes.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// microshift/volume-snapshots-microshift.adoc
+
+:_content-type: CONCEPT
+[id="microshift-storage-classes_{context}"]
+== Storage classes
```

Comment:
In terms of reuse, you want to start this with an H1 and then use the `[leveloffset=+2]` in the assembly to get the correct information architecture.

Reply:
+1